src/ngx_http_datadog_module.cpp

```diff
@@ -709,40 +709,31 @@ static void *create_datadog_loc_conf(ngx_conf_t *conf) noexcept {
     return nullptr;  // error
   }
 
-#ifdef WITH_RUM
-  loc_conf->rum_enable = NGX_CONF_UNSET;
-  loc_conf->rum_snippet = nullptr;
```

Comment:
Why ? Because it is useless (set null by default I guess) ?

Reply:
To be coherent, I now use class member initialization. Check src/datadog_conf.h.